Index: api/id_admin.pub
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/api/id_admin.pub b/api/id_admin.pub
new file mode 100644
--- /dev/null	(date 1688560701716)
+++ b/api/id_admin.pub	(date 1688560701716)
@@ -0,0 +1,1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCnFJx90z8A2isjY7VQpGHsaEMzdSH7Yf8pZQcK2cP3WOXJFrog39U7e17/itrL7rI2QTcZ3PTLPWWIR5JXzsaULGEaYofD60/PVQaqYkgAvHTp8llhZW0V2+tzar02gyUPruuqPKwWSEMpPhuxsms366m9CMpW2X0/zxcnUGdH4bAFi483+hAtUkRhmH5MpxmLZoXBecvc8ybCJPsiYgSaY8eR0vrcXirWwTcd65DvMcCgY6QS38SZhsUxkq1pRgnSy1yy4KJ7670esLub3kplJCLAeotx7W/f7serwYAcdSqRU8y97GS3aaKGZAwX1HLW9diGCpdWx3/dNFFicvSwHCKER0T29r6jHIyg7GpTETnIo3v74Uexmg0wBjEKE6ZkfLJm7mPWvAHGRPyFuPIF8tFDDi+CFziAf8b1cq63Qmjlc1DZZNhywXvgd3PuKN/WqDyP7hfTDi7BFhy7TjVzwPms6GSERfwDtYY5ZeydjptWCAevuNpwD4CqH/wURE8= etienne@MacBook-Pro-de-Etienne.local
